plugins/woocommerce/client/legacy/js/admin/meta-boxes-product-variation.js

```diff
@@ -54,14 +54,6 @@ jQuery( function ( $ ) {
 				var new_attribute_data = $(
 					'.woocommerce_variation_new_attribute_data'
 				);
-				var attribute_name = new_attribute_data.find( 'input[name^="attribute_names"]' ).val();
-				var attribute_value = new_attribute_data
-					.find( 'textarea[name^="attribute_values"]' )
-					.val();
-
-				if ( ! attribute_name || ! attribute_value ) {
-					return;
-				}
```

Comment:
I removed this validation in [this commit](https://github.com/woocommerce/woocommerce/pull/37046/commits/e7833e16af4132d8f4c9fc7bfdc415ce8d7055a2) because we're already disabling the button and it would duplicate the validation.
I'm not opposed to being more defensive, but I wanted a second opinion.